components/brave_ads/core/internal/common/strings/string_conversions_util.cc

```diff
@@ -5,6 +5,9 @@
 
 #include "brave/components/brave_ads/core/internal/common/strings/string_conversions_util.h"
 
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/string_split.h"
```

Comment:
Already included in the header. Please run clang static analyzer across all changed files in this PR. Also you can additionally run `npm run presubmit` to pick up other issues.